src/Microsoft.AspNet.Razor.Runtime/TagHelpers/HtmlAttributeNotBoundAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// Indicates the associated <see cref="ITagHelper"/> property should not be bound to HTML attributes.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = false)]
+    public sealed class HtmlAttributeNotBoundAttribute : Attribute
```

Comment:
This name is a bit of a mouthful. Maybe `NonHtmlAttributeAttribute` to be more consistent with https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/NonControllerAttribute.cs#L13? (Yes, I know I'm suggesting `XyzAttributeAttribute` but we wouldn't be the first ones to do that, and it makes sense.)

@DamianEdwards thoughts on name?